view-admin/src/styles/Home.module.css

```
@@ -174,6 +174,16 @@
   color: #fff;
 }
 
+.dummyButton{
```

```suggestion
.notHoberButton{
```
`dummy`のような抽象的な表現での命名付けは基本良くないです。どういうcssか分からなくなるので。。。
私がつけた名前も一例なので考えてみて～

めっちゃてきとうに見つけた参考文献貼っときます
https://qiita.com/kenogi/items/1321b515fbd433644819